docs/using-airbyte/mappings.md

```diff
@@ -0,0 +1,107 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Mapping fields
+
+Use mappings to hash, encrypt, and rename fields, and filter rows. You set up mappings on each stream, ensuring your source data arrives in your destination exactly as you want it.
+
+## More about mapping in Airbyte
+
+It’s often the case that you want to move data from a source to a destination in a non-literal way, obscuring sensitive information and improving the consistency and usability of that data in its final destination. Mapping allows you to match a field from your source to your destination and sync data in a way that is still accurate, but also more meaningful and appropriate for your needs.
+
+Several types of mapping are possible in Airbyte, and you can combine them together in meaningful ways.
+
+### Hash
+
+Hashing is an **irreversible** process that protects sensitive data by obscuring it. Airbyte supports MD5, SHA-256, and SHA-512 hashing methods. 
```

Comment:
Technically we also support MD2, SHA-1, and SHA-384.  

However, these are only configurable via the API because we didn't want to overwhelm the UI with options that are not used often.  Not sure if that belongs in the docs or just as something folks discover in the API docs themselves.